kuma/wiki/jinja2/wiki/translate.html

```diff
@@ -93,10 +93,10 @@ <h3 class="approved-title">{{ _('Approved %(locale)s version:', locale=default_l
                     </div>
                 </header>
                 <div class="boxed translate-rendered text-content">
-                  {{ based_on.content_cleaned|safe }}
+                  {{ based_on.content|safe }}
```

Comment:
Similar to the revision view, we can't trust old content. Translations may be way out of date, so ``based_on`` could be ancient.

Reply:
Again, good catch, and sorry I missed this.